frontend/model/state.js

```diff
@@ -6,21 +6,21 @@
 import sbp from '~/shared/sbp.js'
 import Vue from 'vue'
 import Vuex from 'vuex'
-import { GIMessage } from '~/shared/GIMessage.js'
-import * as _ from '@utils/giLodash.js'
 import L from '@view-utils/translations.js'
 import currencies from '@view-utils/currencies.js'
 import incomeDistribution from '@utils/distribution/mincome-proportional.js'
+import { GIMessage } from '~/shared/GIMessage.js'
 import { SETTING_CURRENT_USER } from './database.js'
 import { ErrorDBBadPreviousHEAD, ErrorDBConnection } from '~/shared/domains/gi/db.js'
-import { LOGIN, LOGOUT, EVENT_HANDLED, CONTRACTS_MODIFIED } from '@utils/events.js'
 import Colors from './colors.js'
 import { TypeValidatorError } from '@utils/flowTyper.js'
 import { GIErrorUnrecoverable, GIErrorIgnoreAndBanIfGroup, GIErrorDropAndReprocess } from './errors.js'
 import { STATUS_OPEN, PROPOSAL_REMOVE_MEMBER } from './contracts/voting/proposals.js'
 import { VOTE_FOR } from '@model/contracts/voting/rules.js'
 import { actionWhitelisted, CONTRACT_REGEX } from '@model/contracts/Contract.js'
 import { currentMonthTimestamp } from '@utils/time.js'
+import * as _ from '@utils/giLodash.js'
```

Comment:
Do we need every function to lodash? Maybe we should add a comment to import only the necessary functions (sorry for out of context)

Reply:
We use a large number of functions in this file so it's OK to include `_`. The project uses everything defined in `giLodash.js` so tree shaking isn't a problem here